services/deployment-agent/startup_script.bash

```diff
@@ -70,7 +70,7 @@ set +o allexport
 #
 cd "$repo_basedir"
 #
-python -c "import urllib.request,os,sys,urllib; f = open(os.path.basename(sys.argv[1]), 'wb'); f.write(urllib.request.urlopen(sys.argv[1]).read()); f.close();" https://github.com/mikefarah/yq/releases/download/v4.29.2/yq_linux_amd64
+python3 -c "import urllib.request,os,sys,urllib; f = open(os.path.basename(sys.argv[1]), 'wb'); f.write(urllib.request.urlopen(sys.argv[1]).read()); f.close();" https://github.com/mikefarah/yq/releases/download/v4.29.2/yq_linux_amd64
```

Comment:
same comment as above applies

Reply:
ditto

Reply:
I reverted the changes. Now, we use `python` in these 2 scripts